include/wlr/types/wlr_keyboard_group.h

```diff
@@ -18,6 +18,8 @@ struct wlr_keyboard_group {
 	struct wlr_input_device *input_device;
 	struct wl_list devices; // keyboard_group_device::link
 	struct wl_list keys; // keyboard_group_key::link
+	bool destroying;
```

Comment:
Instead of having a `destroying` field, can we initiialize `nrefs` to 1 and make `wlr_keyboard_group_destroy` call `keyboard_group_unref`?